build.gradle

```diff
@@ -34,7 +34,7 @@ subprojects {
     ext['netty.version'] = '4.1.31.Final'
     ext['netty-boringssl.version'] = '2.0.18.Final'
     ext['hdrhistogram.version'] = '2.1.10'
-    ext['mockito.version'] = '2.23.0'
+    ext['mockito.version'] = '2.25.1'
```

Comment:
don't change the version - do that in a different pull request

Reply:
oh, right